<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="leftYAxis">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>26</width>
      <height>300</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="leftXAxis">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>300</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rightYAxis">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>26</width>
      <height>300</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rightXAxis">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>300</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDial" name="leftThrottle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="rightThrottle">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>150</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="axisS1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="axisS2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSA">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
